aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -707,6 +707,14 @@ bool can_use_mem_efficient_attention(sdp_params const& params, bool debug) {
   }
 
 #ifdef USE_ROCM
```

Comment:
@jithunnair-amd Actually I've question about this guard. This logic also applies to CUDA platform.
But let's make it ROCM only since it would be more difficult to get it reviewed as common code path.